src/llmq/quorums_blockprocessor.cpp

```diff
@@ -379,7 +379,7 @@ uint256 CQuorumBlockProcessor::GetQuorumBlockHash(Consensus::LLMQType llmqType,
     int quorumStartHeight = nHeight - (nHeight % params.dkgInterval);
     uint256 quorumBlockHash;
     if (!GetBlockHash(quorumBlockHash, quorumStartHeight)) {
-        return uint256();
+        return {};
```

Comment:
Whats your reasoning for all the `uint256() -> {}`?

Reply:
Wait thats a dump question probably as this it a clang-tidy cleanup, but still.. really? Thats something we should follow? 😆 I don't like it haha.. but.. OK.